react/__mocks__/vtex.store-resources/OrderFormContext.js

```diff
@@ -1,5 +1,85 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import OrderForm from 'orderForm'
 
-export function orderFormConsumer(Comp) {
-  return Comp
+export const contextPropTypes = PropTypes.shape({
```

Comment:
we should make a `@types/vtex-io` and export this stuff there, then, import it here...
my 2 cents